addons/mail/controllers/thread.py

```diff
@@ -163,3 +164,32 @@ def mail_message_update_content(self, message_id, body, attachment_ids, attachme
 
     def _is_message_editable(self, message, **kwargs):
         return message.sudo().is_current_user_or_guest_author or request.env.user._is_admin()
+
+    @http.route("/mail/message/get_followers", methods=["POST"], type="json", auth="public")
+    def message_get_followers(self, id, thread_id, thread_model, limit=20, offset=0):
+        domain = [
+            ("res_id", "=", thread_id),
+            ("res_model", "=", thread_model),
+            ("partner_id", "!=", request.env.user.partner_id.id),
+        ]
+        followersCount = request.env["mail.followers"].search_count(
+                    [("res_id", "=", thread_id), ("res_model", "=", thread_model)]
+                )
+        self_follower = request.env["mail.followers"].search(
+            [
+                ("res_id", "=", thread_id),
+                ("res_model", "=", thread_model),
+                ["partner_id", "=", request.env.user.partner_id.id],
+            ]
+        )
+        data = request.env["mail.followers"].search(domain, offset=offset, limit=limit, order="name ASC")
+        FollowerListView = {
+                "id": id,
+                "followersCount": followersCount,
+                "followers": Store.many_ids(data, "ADD"),
+                "selfFollower": Store.one_id(self_follower),
+        }
+        return {
+            "FollowerListView": FollowerListView,
+            **Store(data).get_result()
+        }
```

Comment:
What we usually do is we directly return the response of `get_result()`. So, what you could do is initiate store with the `data` and then add `FollowerListView` into the store.